### ISSUE TITLE
Import of events broken in 2018.08.1 ?

### ISSUE THREAD

**Issue by ignoreigor** [tine20/tine20#6891]
When I try to import events from a ics-file via the web-frontend, the dialog is not closed and the progress-circle rotates infinitely. I see no errors in the webserver's logfile, even in tine.log (level 8) there is no indication of an error, but JS-console says 

TypeError: this.containerField is undefined[Weitere Informationen]  index.php:1:3760492
	Tine.Calendar.ImportDialog<.doImport https://server.de/tine20/index.php:1:3760492
	Tine.Calendar.ImportDialog<.getPanel/<.onFinishButton< https://server.de/tine20/index.php:1:3768426
	.createDelegate/< https://server.de/tine20/library/ExtJS/adapter/ext/ext-base.js:7:5901
	Tine.widgets.dialog.WizardPanel<.onFinishButton https://server.de/tine20/index.php:1:823411
	Ext.Button<.onClick https://server.de/tine20/library/ExtJS/ext-all.js:14:64244
	C https://server.de/tine20/library/ExtJS/ext-all.js:7:26578

Maybe you can have a look on it.

**Comment by pschuele** [comment]
ok, will have a look - thanks for reporting!

**Comment by sbaumgartner75** [comment]
Maybe related: On the command line it fails as well:

``$ php htdocs/tine20.php --username username --password secret --method Calendar.Import plugib=Calender_Import_Ical importContainerId=26 ical.ics


PHP Parse error:  syntax error, unexpected '->' (T_OBJECT_OPERATOR) in /www/office.example.com/htdocs/Tinebase/Frontend/Cli.php on line 985
``

**Comment by sbaumgartner75** [comment]
Temporarily fixed the problem by changing lines 984 - 985 to:
`        if (! $queueConfig->{Tinebase_Config::ACTIONQUEUE_ACTIVE}) /* ||
                !($actionQueue = Tinebase_ActionQueue::getInstance())->hasAsyncBackend())*/ {
 `

**Comment by pschuele** [comment]
ah, looks like your php version does not support this construct: 

```php
($actionQueue = Tinebase_ActionQueue::getInstance())->hasAsyncBackend()
```
seems like an easy fix...

**Comment by pschuele** [comment]
but I'm afraid, this does not fix @ignoreigor 's problem... :(

**Comment by sbaumgartner75** [comment]
Hmm ... 
`ii  php5                              5.6.36+dfsg-0+deb8u1              all          server-side, HTML-embedded scripting language (metapackage)`
Based on the requirements in the Readme of this repository, PHP 5.6 should be supported ...

**Comment by pschuele** [comment]
hmm, maybe your CLI has a different php version?
please run
```
php --version

**Comment by ignoreigor** [comment]
Oh, you're busy here. I also tried the import via CLI before. There was no problem for me when doing it.

**Comment by sbaumgartner75** [comment]
`$ php --version
PHP 5.6.36-0+deb8u1 (cli) (built: Jun 26 2018 17:31:29)
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
`

**Comment by pschuele** [comment]
ok, thanks for checking. fix for @sbaumgartner75 's problem is coming tomorrow.
I'll try to reproduce @ignoreigor 's problem today.

**Comment by schmidi2** [comment]
The problem reported by @ignoreigor also occurred in my installation. It exists since releases 2018.08.1 and also in the newest release 2018.08.2. 

To solve the issue I downgraded to 2018.02.7:
1. Unpacked 2018.02.7 zip file.
2. Disabled Caching in setup.php, truncated the cache directory.
2. In Application Manager uninstalled HumanResources, the Calendar (version 12.3).
3. Installed application Calendar (version 12.2).
Now I can use the Calendar Import functionality again.

The problem exist because the form has changed in commit #1e3e818d035994654803c948ad3a2e38c052f081: In the General Settings the textfield to the left of the dropdown was removed. This fires the JavaScript error "TypeError: this.containerField is undefined". It was forgotten to remove the reference to the deleted file in Calendar/js/ImportDialog.js . @pschuele
